widget/button.go

```diff
@@ -341,7 +357,7 @@ func (b *buttonRenderer) buttonColor() color.Color {
 	switch {
 	case b.button.Disabled():
 		return theme.DisabledButtonColor()
```

Comment:
A button with low importance has the same background as normal content and no surrounding shadow.
When deactivating it, it suddenly becomes more visible because it has now a background different from the normal content.
However, I think this should be fixed by another PR.